health/README.md

```diff
@@ -1,55 +1,37 @@
 <!--
 ---
 title: "Health monitoring"
+date: 2020-04-24
 custom_edit_url: https://github.com/netdata/netdata/edit/master/health/README.md
 ---
 -->
 
 # Health monitoring
 
-With Netdata, you can monitor the health and performance of your systems and applications. You start with hundreds of
-alarms that have been pre-configured by the Netdata community, but you can write new alarms, tune existing ones, or
-silence any that you're not interested in. 
+The Netdata Agent is a health watchdog for the health and performance of your systems, services, and applications. We've
+worked closely with our community of DevOps engineers, SREs, and developers to define hundreds of production-ready
+alarms that work without any configuration.
 
-Netdata creates charts dynamically, and runs an independent thread to constantly evaluate them, which means Netdata
-operates like a health watchdog for your services and applications.
+The Agent's health monitoring system is also dynamic and fully customizable. You can write entirely new alarms, tune the
+community-configured alarms for every app/service [the Agent collects metrics from](/docs/agent/collectors/collectors/),
```

Comment:
I was reviewing now only the `docs/` directory that are cited in the documentation, and I found this reference that is also not present in the `master` repo.

Reply:
Fixed.